### DISCUSSION TITLE
How to create a new repository

### DISCUSSION THREAD

**Discussion by rajshresth0** [Andipstha/drawIO#4] [Q&A] [answered]
What happens when I call git remote set-url origin just after git init? Does git remote set-url origin create origin? If origin already exists after git init there is no difference between using those commands in my scenario, right?

**Comment by Andipstha** [comment, marked as answer]
below is used to add a new remote:

git remote add origin git@github.com:User/UserRepo.git
below is used to change the url of an existing remote repository:

git remote set-url origin git@github.com:User/UserRepo.git
below will push your code to the master branch of the remote repository defined with origin and -u let you point your current local branch to the remote master branch:

git push -u origin master

**Comment by Andipstha** [comment]
below will reinitialize your local repo; also clear remote repos (ie origin):

git init
Then below, will create 'origin' if it doesn't exist:

git remote add origin [repo-url]
Else, you can use the set-url subcommand to edit an existing remote:

git remote set-url origin [repo-url]
Also, you can check existing remotes with

git remote -v
Hope this helps!
